libproxy/cmake/modules/pacrunner_duktape.cmk

```diff
@@ -1,11 +1,19 @@
+option(WITH_DUKTAPE "Search for Duktape package" ON)
+
 if(WIN32)
-  set(DUKTAPE_FOUND 0)
+  set(DUKTAPE_FOUND 0)  
+  if(WITH_DUKTAPE)
+    find_package(duktape QUIET)
+    if(DUKTAPE_FOUND)
+      include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${DUKTAPE_INCLUDE_DIRS})
+      link_libraries(${DUKTAPE_LIBRARY})
```

Comment:
Ugh, more global definitions. Ideally, we would use `target_include_directories` and `target_link_libraries` at the place of use. But looks like it is also currently used this way by the Unix code so :woman_shrugging:.